src/pages/app/menus/AddonsMenu.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+  import Menu from "@/common/Menu";
+  import MenuItem from "@/common/MenuItem";
+  import AddonMenuItem from "./AddonMenuItem";
+
+  import { layout, newProject } from "@/manager/layout";
+  import { addons } from "@/manager/addons";
+  import { _ } from "svelte-i18n";
+
+  function sort(addons) {
+    if (addons == null) return [];
+    try {
+      addons.sort((a, b) => a.name.localeCompare(b.title));
+    } catch (e) {}
+    return addons;
+  }
+
+  $: alphabetizedAddons = sort($addons.addons);
+</script>
+
+<style lang="scss">
+  .info {
+    color: $gray;
+    font-style: italic;
+    font-size: $small;
+  }
+</style>
+
+<Menu>
+  <MenuItem primary={true} on:click={newProject}>
+    {$_("addonsMenu.newAddon")}
```

Comment:
Since there is no way to add new add-ons for now, I would just remove this menu item.